hazelcast/src/main/java/com/hazelcast/config/DiscoveryStrategyConfig.java

```diff
@@ -33,7 +33,7 @@
  */
 public class DiscoveryStrategyConfig implements IdentifiedDataSerializable {
     private String className;
-    private DiscoveryStrategyFactory discoveryStrategyFactory;
+    private transient DiscoveryStrategyFactory discoveryStrategyFactory;
```

Comment:
maybe also a small note on this field to point the reason of skipping

Reply:
Good idea, added!